vue/components/ui/atoms/button-color/button-color.vue

```diff
@@ -252,6 +261,12 @@
     width: 32px;
 }
 
+.button-color.button-color-medium ::v-deep .loader > div {
+    height: 28px;
+    margin-left: 2px;
+    width: 28px;
+}

```

Comment:
Same for margin-left, none of the other variants have it.

Reply:
It's something that could be addressed on the other side, removing it